docs/en/base/typography.md

```diff
@@ -6,30 +6,9 @@ title: Typography
 
 All text in Vanilla uses the Ubuntu typeface.
 
-Vanilla's typographic scale has a base font size of 14 pixels (small screens) and a font weight of 300.  At the medium breakpoint, the base font size is 15 pixels, and at the large breakpoint it is 16 pixels.
-
-## Blockquotes and citations
-
-<a href="https://vanilla-framework.github.io/vanilla-framework/examples/base/blockquotes/"
-    class="js-example">
-    View example of the base blockquotes
-</a>
-
-## Small text
-
-<small>This is some &lt;small&gt; text with a font size of 13 pixels.</small>
-
-## Strong text
-
-<strong>This is some strong text</strong>
-
-## Superscripted text
-
-<p>This text is <sup>superscripted</sup></p>
-
-## Subscripted text
-
-<p>This text is <sub>subscripted</sub></p>
+Vanilla's typographic scale has a base font size of 14 pixels (small screens) 
+and a font weight of 300.  At the medium breakpoint, the base font size is 15 
+pixels, and at the large breakpoint it is 16 pixels.
```

Comment:
This is incorrect - it's currently 15px on resolutions less than 768px and 16px on resolutions above 768px.

<img width="521" alt="screen shot 2017-04-06 at 11 56 06" src="https://cloud.githubusercontent.com/assets/505570/24751076/24b063ea-1ac0-11e7-9a91-6d2dbd5bbb41.png">